docker-compose.yml

```diff
@@ -12,4 +12,16 @@ services:
     env_file:
       ./.env
     volumes:
-      - ./todo-app:/app/todo_app 
\ No newline at end of file
+      - ./todo-app:/app/todo_app 
+
+
+  selenium:
+    build: 
+      context: .
+      target: development
+      
+    ports:
+        - 5000:5000
+    volumes:
+        - ./todo-app:/app/todo_app 
+    privileged: true
```

Comment:
I don't think you want this `selenium` service in your docker-compose. It's still targetting development and actually is the same as dev but without .env and with privileged: true? 

If you want to run tests via docker-compose, there's no need to publish a port and there's no need for a mounted volume unless you want to run tests in watch mode (where they re-run after you make changes).

I assume this was something you were experimenting with but didn't finish.